wrapper/src/main/java/software/amazon/jdbc/plugin/cache/CacheMonitor.java

```diff
@@ -137,7 +138,7 @@ static class ClusterHealthState {
 
     ClusterHealthState(String rwEndpoint, String roEndpoint, boolean useSSL, Duration cacheConnectionTimeout,
         boolean iamAuthEnabled, AwsCredentialsProvider credentialsProvider, String cacheIamRegion,
-        String cacheName, String cacheUsername, String cachePassword) {
+        String cacheName, String cacheUsername, String cachePassword, boolean isClusterMode) {
```

Comment:
Does the monitor needs to know whether it is CME or CMD? It just relies on ping right?